app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% content_for :title, 'My Tasks' %>
+
+<div class="content-container mt-3">
+  <% if @tasks.any? %>
+    <table class="table table-striped"
+            data-behavior="dradis-datatable"
+            data-default-columns="<%= @default_columns %>"
+            data-local-storage-key="<%= @local_storage_key %>">
+      <thead>
+        <tr>
+          <th>Title</th>
+          <% unless params[:project_id] %>
+            <th>Project</th>
+          <% end %>
+          <th>Methodology</th>
+          <th>Due Date</th>
+          <th>Assigned</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @tasks.each do |task| %>
+          <tr>
+            <td data-sort="<%= task.name %>">
+              <%= link_to task.name, project_board_list_card_path(task.project, task.board, task.list, task) %>
+            </td>
+            <% unless params[:project_id] %>
```

Comment:
Ditto to the 2-controller comment above

Reply:
Because this is still a basically identical view shared between layouts, I went with the notifications_controller approach which also shares a view between layouts. Happy to change this if you feel strongly about it.